trl/trainer/dpo_trainer.py

```diff
@@ -309,6 +312,7 @@ def make_inputs_require_grad(module, input, output):
         self.padding_value = padding_value
 
         self.beta = beta
+        self.label_smoothing = label_smoothing
```

Comment:
Maybe add a warning if the parameter is set but IPO is used since then it has no effect.